client/src/GameContext.js

```diff
@@ -37,55 +66,55 @@ const GameContextProvider = ({ children }) => {
     {
       name: 'James',
       alive: true,
-      role: 'barista',
+      role: 'none',
```

Comment:
Who will be making this not hardcoded again?

Reply:
I believe Ahnaf when he implements the add player page; I've implemented assigning roles so i "un-hardcoded" that